src/app/pages/CreateConnectorPage/StepReview.tsx

```diff
@@ -57,6 +57,27 @@ export function Review() {
   );
 }
 
+const ConnectorProperties = (
```

Comment:
Cool!  Can't decide if it's big enough to warrant it's own component.  I guess if there's a need to add another case or more logic around this, I would definitely suggest at that point turning this into a component to help simplify the logic more.

Reply:
That's a great idea, thanks!